planning/behavior_path_planner/src/path_utilities.cpp

```diff
@@ -36,14 +36,14 @@ namespace behavior_path_planner::util
 std::vector<double> calcPathArcLengthArray(
   const PathWithLaneId & path, size_t start, size_t end, double offset)
```

Comment:
add const to offset argument

Reply:
DONE [593f95c](https://github.com/autowarefoundation/autoware.universe/pull/1832/commits/593f95c6ae7e1d6e0e29b710bc0b75e19f295570)